### PR TITLE
Fire event on add/remove outcome

### PR DESCRIPTION
**Pull request by rgoessl**

### CODE REVIEW

File: d2l-activity-alignment-tag-list.js [hunk 1]
```diff
@@ -107,6 +107,8 @@ class ActivityAlignmentTagList extends mixinBehaviors([
 	}
 
 	_getAlignmentHrefs(entity) {
+		this.fire('d2l-activity-alignment-outcomes-updated', entity);
```
Comment (xiangqiyan) [inline d2l-activity-alignment-tag-list.js]: I think you can use this.dispatchEvent with a CustomEvent instead of this.fire

Comment (rgoessl) [reply d2l-activity-alignment-tag-list.js]: What's the difference?

Comment (xiangqiyan) [reply d2l-activity-alignment-tag-list.js]: this.fire is just a legacy api from polymer 1. dispatchEvent is now the standard for firing custom events

Comment (rgoessl) [reply d2l-activity-alignment-tag-list.js]: I'll change it

**Pull request merged**